pkg/util/iptables/iptables.go

```diff
@@ -192,12 +192,6 @@ const WaitString = "-w"
 // WaitSecondsValue a constant for specifying the default wait seconds
 const WaitSecondsValue = "5"
 
-// WaitIntervalString a constant for specifying the wait interval flag
```

Comment:
`WaitIntervalMinVersion` line 180 can also be removed but this can be a follow up, seems we are all set now

/lgtm